app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,8 @@
+module SessionsHelper
+
+    # Logs in the given user.
+    def log_in(user)
+      #session[:user_id] = user.id
+      cookies.encrypted[:user_id] = user.id
+    end
+  end
```

Comment:
This is code that we should only run in a select view controllers. Helpers are for views, but putting it in a module is a good practice anyway. I would move this code over to the `Authentication` concern in app/controllers/concerns/authentication.rb.